dataRetriever/requestHandlers/requestHandler.go

```diff
@@ -551,6 +552,36 @@ func (rrh *resolverRequestHandler) RequestMetaHeaderByNonce(nonce uint64) {
 	rrh.addRequestedItems([][]byte{key}, uniqueMetaHeadersSuffix)
 }
 
+// RequestValidatorInfo asks for the validator info associated with a specific hash from connected peers
+func (rrh *resolverRequestHandler) RequestValidatorInfo(hash []byte) {
+	log.Debug("requesting validator info messages from network",
```

Comment:
You can define in the beginning of the file this identifier:

`const uniqueValidatorInfoSuffix = "vi"`

and add here this check: 

	if !rrh.testIfRequestIsNeeded(hash, uniqueValidatorInfoSuffix) {
		return
	}

then in L573 we will whitelist this hash requested:

`rrh.whiteList.Add([][]byte{hash})`
	
and before L583 we will add this request to avoid another one done with the same hash in the next 1 sec:

`rrh.addRequestedItems([][]byte{hash}, uniqueValidatorInfoSuffix)`

This way we will protect also the validator info request for many requests in a short time frame and also we will whitelist these requests to be accepted on the interceptor in any situation.